### ISSUE TITLE
Zuordnung Unterveranstaltung zu übergeordneten (bspw. mehrtägige Festivals)

### ISSUE THREAD

**Issue by alexanderkoenitz** [NicoKnoll/BLNFMCalSync#6]
Es fehlen noch die Verknüpfungsmöglichkeit:
„Mutter“-Event (via ID in Spalte Parent (ID) in Tabelle)

**Comment by nicoknoll** [comment]
Wird als Attribut gespeichert werden, da das Plugin nativ keine Sub-Events unterstützt. (#_ATT{parent})
Alternativ könnte man für die Events + Subevents eine Extrakategorie einführen.

**Comment by alexanderkoenitz** [comment]
ja, bitte als custom field / attribut speichern. (danach kann das related posts-plugin darauf zugreifen und die verknüpfung sichtbar machen)
